Source/Scene/DebugCameraPrimitive.js

```diff
@@ -149,8 +159,33 @@ define([
                 values : positions
             });
 
+            var offset;
+
             // Create the outline primitive
-            var outlineIndices = new Uint16Array([0,1,1,2,2,3,3,0,0,4,4,7,7,3,7,6,6,2,2,1,1,5,5,4,5,6]);
+            var outlineIndices = new Uint16Array(8 * (2 * numFrustums + 1));
+            // Build the far planes
+            for (f = 0; f < numFrustums + 1; ++f) {
+                outlineIndices[f * 8] = f * 4;
```

Comment:
Here and in the loop below, assign `f * 8` and `f * 4` to locals.  I do not know if the JS engine would optimize.  This primitive is just for debugging, but in general `primitive.update` functions are carefully coded for performance.